src/patients/appointments/AppointmentsList.tsx

```diff
@@ -54,6 +54,20 @@ const AppointmentsList = (props: Props) => {
 
   return (
     <Container>
+      <div className="row">
+        <div className="col-md-12 d-flex justify-content-end">
+          <Button
+            outlined
+            color="success"
+            icon="add"
+            iconLocation="left"
+            onClick={() => history.push('/appointments/new')}
+          >
+            {t('patient.appointments.new')}
```

Comment:
Let's use the the existing `New Appointment` label that you can see on the appointments calendar.

let's also add the new appointment icon to the button

Reply:
ok